test-data/run.test

```diff
@@ -2385,3 +2385,12 @@ g()
 [out]
 (((1, 2), 2), 1, 'hi', 'hi2')
 caught the exception
+
+[case forwardReference]
+class A:
+    b: 'B'
+class B: pass
+
+[file driver.py]
+from native import A, B
+A()
```

Comment:
Could you merge this in with some other test in `run-classes.test`, to keep the number of tests down.